### DISCUSSION TITLE
런타임 에러

### DISCUSSION THREAD

**Discussion by TARK-YEJI** [mtak0235/helloPython#8] [Q&A]
a = input("A:")
A = int(a)
b = input("B:")
B = int(b)
print (A+B)

**Comment by TARK-YEJI** [comment]
이 코드를 올렸는데 vscode에서는 터미널에 사용자 입력이 가능하고 결과값이 맞게 출력을 하는데 백준에 올리니까 런타임 에러라고 나옵니다.

**Reply by mtak0235** [reply]
제 생각에는 백준 사이트 문제 제출 페이지에서 컴파일러를 디폴트(c++)에서 python으로 바꾸지 않아 그런 것 같습니다 시도 ㄱㄱ

**Reply by TARK-YEJI** [reply]
어...python으로 되어있습니다

**Reply by TARK-YEJI** [reply]
혹시 코드 자체가 오류가 있는걸까요?

**Reply by mtak0235** [reply]
문제를 다시 확인해보니, 문제에서 명시된 입력의 형태가
```
1
2
```
가 아니라 
```
1 2
```
였습니다. 즉, **한번에 숫자 두개를 입력받는다** 라는 의미이죠. 다시 말해 input()함수가 한번 쓰여야 합니다. 그렇다면 하나의 문자열에 담긴 두개의 숫자를 잘 잘라서 쓰라는게 문제의 의도겠죠? 
그렇다면 런타임 에러는 왜 떴는가? @TARK-YEJI 님이 작성하신 프로그램은 프로그램 사용자(백준의 입력 프로그램)가 두번 입력하기를 기대합니다. 하지만 사용자(백준의 입력프로그램)는 위의 두번째 케이스로 한번밖에 입력해주지 않죠. 즉 님이 작성하신 프로그램은 처음 입력을 방아서 처리한 후, 두번쨰 입력을 받으려고 기다리고 있지만 사용자는 두번째 입력을 줄 생각이 없죠.("응 이미 숫자 두개 다 줬잖아 돌아가") 고로 니이 작성하신 프로그램은 입력을 영원히 기다리고 사용자는 영원히 입력을 줄 생각이 없습니다. 
이떄 발생하는것으로 추정되는 것은 time out 입니다. 그래서 이게 런타임 에러의 형태로 나온게 아닐까 생각이 듭니다.

**Reply by TARK-YEJI** [reply]
고작 a+b인데...ㅠ 나는 왜 이런가

**Reply by mtak0235** [reply]
갱차나 a+b보다 더 어려운거였어

**Comment by mtak0235** [comment]
추가로 코멘트를 드리자면, input에 매개변수를 넣으셨는데, 사용자의 이해를 돕기 위한 의도라는 것은 알겠으나, 문제에서 요구하는 입출력 양식과 맞지 않습니다. 입출력 형식을 맞춰주세요(여기서는 `input()`매개변수를 지우면 되겠죠?)
